.github/workflows/validation.yml

```diff
@@ -0,0 +1,37 @@
+name: Validation
+
+on: pull_request
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install dependencies
+        run: |
+          npm install
+      - name: ESLint
+        run: npm run lint
+
+  test:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install dependencies
+        run: npm install
+      - name: Mocha
+        run: npm run test --coverage
+      - name: Coveralls
+        uses: coverallsapp/github-action@v1.1.2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Can you switch us to codecov similar to `graphql-js`?
Codecov doesn't require any tokens for public repos.

Reply:
oh, I saw coveralls somewhere in the project. sure thing.